src/masonite/drivers/broadcast/BroadcastAblyDriver.py

```diff
@@ -52,8 +52,9 @@ def channel(self, channels, message, event='base-event'):
             raise DriverLibraryNotFound(
                 'Could not find the "ably" library. Please pip install this library running "pip install ably"')
 
+        configuration = config('broadcast.drivers.ably')
         client = AblyRest('{0}'.format(
-            self.config.DRIVERS['ably']['secret']
+            configuration['secret']
```

Comment:
The conclusion is to do 

```python
        configuration = config('broadcast.drivers.ably')
        if not configuration:
            raise Exception('Could not find ably broadcast configuration')

        client = AblyRest('{0}'.format(
            self.config.DRIVERS['ably']['secret']
            configuration['secret']
```

Reply:
okay will check it once and will raise the PR, will update this same PR tomorrow :)

Reply:
done. please reivew :)